### ISSUE TITLE
[FEATURE REQUEST] Allow auto upload of more than just images and videos, without a limit of one per

### ISSUE THREAD

**Issue by IveMalfunctioned** [owncloud/android#3895]
**Is your feature request related to a problem? Please describe.**
Currently, the ownCloud app only allows for auto uploading one directory of images and one directory of videos at once. It's very limiting and also not very clear about what file types it will upload. I also want to be able to upload more than just images and videos and more than one directory automatically.

**Describe the solution you'd like**
It would be nice if the app could allow for adding many custom directories in a list and adding custom file types (or all file types) for each one.

**Comment by jesmrec** [comment]
This is related with https://github.com/owncloud/android/issues/3126 and other issues... interesting to have, it will be like a local folder sync action.

**Comment by michaelstingl** [comment]
> it will be like a local folder sync action.

I don't read bi-directional sync. Only **upload** of newly added files in multiple folders.

**Comment by jesmrec** [comment]
yes, this is the idea. The issue i mentioned suggests to get any folder to back up... this is not exactly the same (adding only new items is the requested here) but both should be one-direction-driven (device -> cloud).
